Field Base: Select

astrit opened the pull request.

splashdust reviewed (changes requested):
Overall it looks super good! Added a few nit picks in the review comments.

Additionally, there seem to be some events related issue. It seems like the value is 'lagging behind' and returning the previously selected one. I added it to the React and Angular examples, so you can have a look at it there in the context of a form with other components.

And one last thing, since this is a brand new component, it would be good to mark it a beta!

astrit commented on libs/core/src/components/select/select.ts:
```diff
@@ -0,0 +1,215 @@
+  static styles = [tokens, styles]
+
+  @property()
+  placeholder = ''
```
The value issue has been fixed now! 
Placeholder removed as I agree it should not be there.

astrit commented:
> Overall it looks super good! Added a few nit picks in the review comments.
> 
> Additionally, there seem to be some events related issue. It seems like the value is 'lagging behind' and returning the previously selected one. I added it to the React and Angular examples, so you can have a look at it there in the context of a form with other components.
> 
> And one last thing, since this is a brand new component, it would be good to mark it a beta!

Thank you very much for tremendous feedback! 
As noted all the previous issues have been fixed including the value! 
The beta status was there all along not sure if is missing somewhere else ? 
I improved a bit the documentation should be good enough!
